oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsNetworkIF.java

```diff
@@ -187,6 +194,7 @@ public boolean updateAttributes() {
             this.inDrops = ifRow.InDiscards; // closest proxy
             this.speed = ifRow.ReceiveLinkSpeed;
             this.ifAlias = Native.toString(ifRow.Alias);
+            this.ifOperStatus = IfOperStatus.byValue(ifRow.OperStatus);
```

Comment:
It looks to me like the Windows value here is 0-indexed while the `byValue()` method assumes 1-indexed. Can you test/confirm?

Reply:
Actually the Windows ones are properly indexed, so no problem here.  From the SDK:
```
//
// OperStatus values from RFC 2863
//
typedef enum {
    IfOperStatusUp = 1,
    IfOperStatusDown,
    IfOperStatusTesting,
    IfOperStatusUnknown,
    IfOperStatusDormant,
    IfOperStatusNotPresent,
    IfOperStatusLowerLayerDown
} IF_OPER_STATUS;
```

Reply:
Yes, i checked here: [https://docs.microsoft.com/en-us/windows/win32/api/netioapi/ns-netioapi-mib_if_row2](https://docs.microsoft.com/en-us/windows/win32/api/netioapi/ns-netioapi-mib_if_row2).

Also did some testing on my machine. At least UP and DOWN are mapped correctly.